server/db/mongodb/adapter.go

```diff
@@ -1955,7 +1963,7 @@ func (a *adapter) MessageDeleteList(topic string, toDel *t.DelMessage) error {
 
 // MessageGetDeleted returns a list of deleted message Ids.
 func (a *adapter) MessageGetDeleted(topic string, forUser t.Uid, opts *t.QueryOpt) ([]t.DelMessage, error) {
-	var limit = a.maxResults
+	var limit = a.maxMessageResults
```

Comment:
It should be left as `maxResults`. Deleted ranges are collapsed into a single `meta` packet.

Reply:
Done.